Hearthstone Deck Tracker/Importing/CardImageImporter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Hearthstone_Deck_Tracker.Utility.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hearthstone_Deck_Tracker.Importing
+{
+	public static class CardImageImporter
+	{
+		const string imageurl = "https://art.hearthstonejson.com/v1/256x/SCH_614.jpg";
+		const string HearthstoneArtUrl = "https://art.hearthstonejson.com/v1/256x";
+		private static string StoreImagesPath = Config.AppDataPath + "\\Images";
+
+		static CardImageImporter()
+		{
+			CheckForAndAddImageFolder();
```

Comment:
You can just inline all of this
```suggestion
			if(!Directory.Exists(StoreImagesPath))
				Directory.CreateDirectory(StoreImagesPath);
```